### PR TITLE
fix(kibbeh): add formatted title in profile pages

### LINKED ISSUE #2442: Missing formated title in profile pages
HTML title when in profile pages displays "https://dogehouse.tv/u/username".

1. Go to any profile page dogehouse.tv/u/username
2. look at title of the tab
3. See error

**Expected behavior**
Title that looks somewhat like "u/username | DogeHouse" like on the dashboard page.

**What device are you on?**
Windows 10 Home 19042.928
Brave Browser Version 1.23.75 Chromium: 90.0.4430.93 (Official Build) (64-bit

### PR DESCRIPTION
**Pull request by thuaung30**
From: url/u/username

To: username | DogeHouse

Screen Shot 2021-04-30 at 19 40 22

fix #2442

### CODE REVIEW

File: kibbeh/src/modules/user/UserPage.tsx [hunk 1]
```diff
@@ -11,6 +12,11 @@ export const UserPage: PageComponent<UserPageProps> = ({}) => {
   const { query } = useRouter();
   return (
     <DefaultDesktopLayout>
+      <HeaderController
+        title={
+          query.username && !Array.isArray(query.username) ? query.username : ""
```
Comment (amitojsingh366) [inline kibbeh/src/modules/user/UserPage.tsx]: why are you checking if the username is not an array?

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: kibbeh/src/modules/user/UserPage.tsx (hunk 1 above)
Comment (thuaung30) [reply kibbeh/src/modules/user/UserPage.tsx]: > why are you checking if the username is not an array?

The type of the query.username is string | string[] | undefined, so I added it just to make sure I don't get the type string[].

File: kibbeh/src/modules/user/UserPage.tsx [hunk 2]
```diff
@@ -11,6 +12,11 @@ export const UserPage: PageComponent<UserPageProps> = ({}) => {
...
+        }
+      />
       <MiddlePanel>
         <UserProfileController
           key={typeof query.username === "string" ? query.username : ""}
```
Comment (Tchoukoualeu) [inline kibbeh/src/modules/user/UserPage.tsx]: Why not use this same logic? `typeof query.username === "string" ? query.username : ""` perhaps, putting it in a `const` so you do not have to write it twice!?

Comment (thuaung30) [reply kibbeh/src/modules/user/UserPage.tsx]: Oh snap, my mind fried and didn't see it, thanks I will change it!

**Comment by amitojsingh366** [comment]
please ensure you are following the conventional commit message format, to fix your latest commit message do:
`git commit --amend -m "refactor(kibbeh): Remove array check and combine the check with UserProfileController"`

**Comment by thuaung30** [comment]
> please ensure you are following the conventional commit message format, to fix your latest commit message do:
> `git commit --amend -m "refactor(kibbeh): Remove array check and combine the check with UserProfileController"`

I'm on it!

**Comment by amitojsingh366** [comment]
@thuaung30 even i didn't know this XD
image

**Comment by thuaung30** [comment]
> @thuaung30 even i didn't know this XD
> image

I was confused when the check failed too,  I changed the commit message again and hopefully, it passes the check this time xD

**Pull request merged**